.github/workflows/kiali-release.yml

```diff
@@ -0,0 +1,117 @@
+name: Kiali Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_type:
+        description: 'Release Type'     
+        required: true
+        default: 'auto' 
+        type: choice
+        options:
+        - auto
+        - minor
+        - snapshot.0
+        - snpashot.1
+        - edge 
+      backend_repo:
+        description: 'The GitHub repo of the back-end sources, in owner/repo format'
+        required: true
+        default: leandroberetta/kiali 
+        type: string
+      quay_name:
+        description: 'The name of the Quay repository to push the release'        
+        type: string
+        default: quay.io/leandroberetta/kiali
+        required: true 
+
+jobs:
+  initialize:
+    name: Initialize
+    runs-on: ubuntu-latest
+    outputs:
+      target_branch: ${{ steps.target_branch.outputs.target_branch }}
+      release_type: ${{ steps.release_type.outputs.release_type || 'auto' }}      
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      # The initialize job gathers variables for later use in jobs.
+      # We are using this technique rather environment variables because at the moment, they won't work with reusable jobs.
+      # A positive side effect of this is that we can print all variables at start for debugging and troubleshooting.
+      - id: target_branch
+        run: echo "::set-output name=target_branch::${{ github.base_ref || github.ref_name }}"        
+      - name: Determine release type
+        id: release_type
+        if: ${{ github.event.inputs.release_type == 'auto'}}
+        run: echo "::set-output name=release_type::$(./.github/workflows/util/release_type.sh)"
+  build_backend:
+    name: Build backend
+    uses: ./.github/workflows/build-backend.yml
+    needs: [initialize]
```

Comment:
this doesn't have `target_branch` unlike the others. Doesn't it need that?

Reply:
That one is not using it. The frontend does use it to compare the new changes with a yarn command (afaik). And the integration tests use it for the kind setup.